packages/material-ui/src/ExpansionPanel/ExpansionPanel.js

```diff
@@ -58,18 +70,7 @@ export const styles = theme => {
       },
     },
     /* Styles applied to the root element if `expanded={true}`. */
-    expanded: {
-      margin: '16px 0',
-      '&:first-child': {
-        marginTop: 0,
-      },
-      '&:last-child': {
-        marginBottom: 0,
-      },
-      '&:before': {
-        opacity: 0,
-      },
-    },
+    expanded: {},
```

Comment:
Follow the pseudo class convention. This state is dynamic and overrides previously defined styles. It should have a specificity of 2.